Werewolf for Telegram/Werewolf Node/Werewolf.cs

```diff
@@ -3723,6 +3744,8 @@ private void NightCycle()
                                 case IRole.Augur:
                                     ConvertToCult(target, voteCult, Settings.AugurConversionChance);
                                     break;
+                                case IRole.ScapeGoat:
+                                    ConvertToCult(target, voteCult, Settings.ScapeGoatConversionChance)
```

Comment:
Really? This poor role has resistance against the cult?
Doesn't feel very logical to me...
At least you added it to the end of the list this time :)